src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,26 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        int ageLimit = 18;
+        int minLengthPassword = 6;
+        if (user == null || user.getAge() == null || user.getLogin() == null
+                || user.getPassword() == null) {
```

Comment:
it's better to have two separate ifs (one for user == null and the second one for fields)